### DISCUSSION TITLE
What github product is right for us? - branch protection

### DISCUSSION THREAD

**Discussion by halldorr** [community/community#29964] [New to GitHub]
We are a small organization and have an organization set up here on Github with private repos. There are just two of us who are actually part of the organization and then we have contributors. One thing we'd like to do is protect various branches. For example, set it up so the branch requires a pull request before a merge, protect from deletions etc. So we're wondering what is best for us to set up here with.

**Comment by NikhilMath** [comment]
Here is a link to set up [Branch Protection Rules](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/managing-a-branch-protection-rule)

Does this help you?

**Reply by IAM8INMIND** [reply]
this is all new i have a great mind for memorization and patterns with numbers writing and certain skillsets but they are scattered i'm a artist creative and sound design specialist and i kinda wanted to transfer into something and just be able to learn more i feel learning sound design could totally be attribute or intagible that is great to have
